run_dir/design/application.html

```diff
@@ -16,7 +16,7 @@ <h2 id="page_title">Projects with application <em>{{ application }}</em></h2><br
 <script>
 $.getJSON("/api/v1/application/{{ application }}", function(data) {
   $.each(data, function(k, project) {
-    $("#project_list").append('<li><a class="text-decoration-none" href="/projects/' + project + '">' + project + '</a></li>')
```

Comment:
This part is a fix for the broken links right? 👍

Reply:
Yes!